apps/example/src/config/equinox.ts

```diff
@@ -54,8 +56,9 @@ export namespace MessageDb {
     return MessageDb.createCached(name, events, fold, access, config)
   }
 
-  export function createProjected<E, S, C>(name: string, events: Events<E, C>, fold: Fold<E,S>, config: { context: MessageDbContext; cache: ICache }) {
-    const access = AccessStrategy.AdjacentSnapshots
+  export function createProjected<E, S, C>(name: string, events: Events<E, C>, fold: Fold<E,S>, project: (client: Client, streamId: StreamId, state: S) => Promise<void>, sub: AccessStrategy<E, S> | undefined, config: { context: MessageDbContext; cache: ICache }) {
```

Comment:
can you mix the project function into each case?
If not maybe make it an optional arg on the mdb category?

Reply:
That's a good point, I'll give that a try, see if I like the API better that way

Reply:
yeah, but it is largely orthogonal to the access strategy
(but I do get how you want to put there difference in people's faces to make people aware of store/category specific features given that the access strategy is the primary place such features and considerations surface)

Reply:
done